dependencyManagement/build.gradle.kts

```diff
@@ -83,7 +83,7 @@ val DEPENDENCIES = listOf(
   "eu.rekawek.toxiproxy:toxiproxy-java:2.1.7",
   "io.github.netmikey.logunit:logunit-jul:2.0.0",
   "io.jaegertracing:jaeger-client:1.8.1",
-  "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:1.39.0-alpha",
+  "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:1.45.0-alpha",
```

Comment:
you can fix renovate to not forget by adding contrib to:

https://github.com/open-telemetry/opentelemetry-java/blob/e6f90f58ce82c861f91fdf7474914ac66939b28f/.github/renovate.json5#L19-L22